fusor-ember-cli/app/templates/satellite.hbs

```diff
@@ -19,7 +19,7 @@
       {{#link-to 'satellite.access-insights' tagName='li' disabled=disableTabAccessInsights}}
         <a>
           <span class="hanging-indent">
-            1C. Insights
+            1C. Red Hat Insights
```

Comment:
This is s small change that probably could be in a separate commit. I noticed in on SJ's new wireframes.